dotCMS/src/main/java/com/dotcms/publisher/pusher/PushPublisher.java

```diff
@@ -216,73 +215,79 @@ public PublisherConfig process ( final PublishStatus status ) throws DotPublishi
 				for (PublishingEndPoint endpoint : endpoints) {
 					EndpointDetail detail = new EndpointDetail();
 
-					InputStream bundleStream = new BufferedInputStream(Files.newInputStream(bundle.toPath()));
-
+					InputStream bundleStream = new BufferedInputStream(Files.newInputStream(bundleFile.toPath()));
```

Comment:
Codacy found an issue: [Ensure that resources like this BufferedInputStream object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)